apps/frontend/src/components/dashboard/links/link/link.tsx

```diff
@@ -107,6 +107,19 @@ export const LinkBlock = component$(
               />
             </div>
           </div>
+          <div class="gap-4 mt-2 flex items-center justify-between sm:hidden">
+            {expirationTime && (
+              <div class="flex justify-start items-center gap-1 ml-3 ">
+                <span class="text-xs font-medium text-gray-500 mb-1 mt-1">Expire At : </span>
```

Comment:
Please keep the text as the other screens: `Expire At` instead of `Expire At :`

Reply:
Done